### ISSUE TITLE
The argument "region" is required, but was not set

### ISSUE THREAD

**Issue by isvolkoff** [tf-cloud-modules/terraform-sbercloud-networking-secgroup#1]
**Describe the bug**
After "terraform plan":
```
Error: Missing required argument
The argument "region" is required, but was not set.
```
I tried to substitute the region value into the module as well as into each rules dictionary, the result is the same

Terraform version: 1.5.4
sbercloud v1.11.5

Settings:
```
provider "sbercloud" {
  auth_url = "https://iam.ru-moscow-1.hc.sbercloud.ru/v3"
  region = "ru-moscow-1"
}
module "secgroup" {
    source  = "tf-cloud-modules/networking-secgroup/sbercloud"

    name = "sg-test"
    ingress_rules = [
    {
        ports             = 80
        protocol         = "udp"
        description      = "User-service ports"
        remote_ip_prefix = "10.10.0.0/16"
    },
    {
        ports             = 443
        protocol        = "tcp"
        description     = "Traffic from certain sec group"
        remote_group_id = "xxxx-xxx-xxx"
    }
    ]
}
```

Trace:
[ERROR] vertex "provider[\"registry.terraform.io/sbercloud-terraform/sbercloud\"]" error: Missing required argument
[TRACE] vertex "provider[\"registry.terraform.io/sbercloud-terraform/sbercloud\"]": visit complete, with errors

[TRACE] dag/walk: upstream of "module.secgroup.sbercloud_networking_secgroup.this (expand)" errored, so skipping
[TRACE] dag/walk: upstream of "module.secgroup.output.id (expand)" errored, so skipping
[TRACE] dag/walk: upstream of "module.secgroup.local.this_sg_id (expand)" errored, so skipping
[TRACE] dag/walk: upstream of "module.secgroup.sbercloud_networking_secgroup_rule.egress_rules (expand)" errored, so skipping
[TRACE] dag/walk: upstream of "module.secgroup.sbercloud_networking_secgroup_rule.ingress_rules (expand)" errored, so skipping
[TRACE] dag/walk: upstream of "module.secgroup (close)" errored, so skipping
[TRACE] dag/walk: upstream of "provider[\"registry.terraform.io/sbercloud-terraform/sbercloud\"] (close)" errored, so skipping

**Comment by andreilapkin** [comment]
@isvolkoff thanks for reporting! Will try to push a fix first thing next morning.

**Comment by andreilapkin** [comment]
@isvolkoff sorry for the delayed answer. I tried the following and the module works perfect:
```
provider "sbercloud" {
  auth_url = "https://iam.ru-moscow-1.hc.sbercloud.ru/v3"
  region   = "ru-moscow-1"
}

terraform {
  required_version = ">= 0.13.1"

  required_providers {
    sbercloud = {
      source  = "sbercloud-terraform/sbercloud"
      version = ">= 1.9.0, < 2.0.0"
    }
  }
}

module "secgroup" {
    source  = "tf-cloud-modules/networking-secgroup/sbercloud"

    name = "sg-test"
    ingress_rules = [
    {
        ports             = 80
        protocol         = "udp"
        description      = "User-service ports"
        remote_ip_prefix = "10.10.0.0/16"
    },
    {
        ports             = 443
        protocol        = "tcp"
        description     = "Traffic from certain sec group"
        remote_group_id = "8d485ffa-3a2e-44d0-9b84-337b9a412c8b"
    }
    ]
}
```

I'm sure you already checked it , but just in case - ```remote_group_id``` so should be provided with the value of the ID of security group that you are referring to. In this example I'm using one of mine. 

Terraform v1.6.2
sbercloud v1.11.5

**Comment by isvolkoff** [comment]
@andreilapkin Thank you for your response. Possible reason is that Terraform is blocked in Russia, and it is necessary to use [mirrors](https://cloud.ru/ru/docs/terraform/ug/topics/guides__mirrors-terraform-advanced.html) for terraform provider sbercloud instead of directly specifying the source as "sbercloud-terraform/sbercloud". I was able to resolve the issue by cloning the source repository and changing the source "sbercloud-terraform/sbercloud" to the mirror "tf.repo.sbc.space/sbercloud-terraform/sbercloud" in the "version.tf" file. After that, I used the cloned repository as the module source for my configuration. The error was no longer observed, and everything is working fine now.  Thank you for your help.
